<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Islamic Star Patterns</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="GLContainer" name="widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveSVG"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>225</width>
     <height>552</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>225</width>
     <height>524287</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="tilingCheckBox">
       <property name="text">
        <string>Show Tiling</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>Tilings</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Archimedeans</string>
        </property>
        <property name="flags">
         <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>3.3.3.3.3.3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.3.3.3.6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.3.4.3.4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.4.6.4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.3.3.4.4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.6.3.6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.12.12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.4.4.4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.6.12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.8.8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6.6.6</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Hanbury Tilings</string>
        </property>
        <property name="flags">
         <set>ItemIsDragEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>10 RD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8.12 RD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9.12 RD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.4.12 RD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8 RD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4.8.8 RD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3.12.12 RD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>alhambra RD</string>
         </property>
        </item>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="wSpinBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="hSpinBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="angleSpinBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>90</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>45</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Contact Delta</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="contactDeltaSpinBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ribbon Width</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="rWidthSpinBox">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.020000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Contact Angle</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QPushButton" name="starColorButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Star Color</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="3">
      <widget class="QPushButton" name="ribbonColorButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Ribbon Color</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="3">
      <widget class="QPushButton" name="interlacingColorButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Line Color</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="3">
      <widget class="QPushButton" name="backgroundColorButton">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Background Color</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Line Width</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="lineWidthSpinBox">
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>2.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.001000000000000</double>
       </property>
       <property name="value">
        <double>0.006000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSaveSVG">
   <property name="text">
    <string>Save SVG</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLContainer</class>
   <extends>QWidget</extends>
   <header>GLContainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
